modules/aws/vpn/vpn.tf

```diff
@@ -65,12 +73,12 @@ resource "aws_route" "vpc_route_0" {
   count                  = "${var.aws_customer_gateway_id_0 == "" ? 0 : 2}"
   route_table_id         = "${var.aws_private_route_table_ids[count.index]}"
   destination_cidr_block = "${var.aws_external_ipsec_subnet_0}"
-  gateway_id             = "${aws_vpn_gateway.vpn_gw.id}"
+  gateway_id             = "${aws_vpn_gateway.vpn_gw[0].id}"
 }
 
 resource "aws_route" "vpc_route_1" {
   count                  = "${var.aws_customer_gateway_id_1 == "" ? 0 : 2}"
   route_table_id         = "${var.aws_private_route_table_ids[count.index]}"
   destination_cidr_block = "${var.aws_external_ipsec_subnet_1}"
-  gateway_id             = "${aws_vpn_gateway.vpn_gw.id}"
+  gateway_id             = "${aws_vpn_gateway.vpn_gw[0].id}"
```

Comment:
`0` or `1`?
```suggestion
  gateway_id             = "${aws_vpn_gateway.vpn_gw[1].id}"
```

Reply:
No, it should be `0` here. `vpn_gw` has only one resource. But new terraform explicitly asks for list references. So if vpn vpc route should be created - it will always have same gateway.